helm/hpcc/templates/dali.yaml

```diff
@@ -68,6 +68,8 @@ spec:
 {{- end }}
 {{ include "hpcc.addSecretVolumes" (dict "root" $ "categories" (list "system" ) ) | indent 6 }}
 ---
+{{- $storage := ($.Values.storage | default dict) -}}
+{{- $daliStorage := ($storage.daliStorage | default dict) }}
```

Comment:
should this default to the first plane with a dali label?  And default dlls/data similarly?

Reply:
I wanted to consider getting rid of daliStorage (and prob. dllStorage too),
but thought I'd not require it yet, and leave for separate PR.